src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java

```diff
@@ -111,22 +104,22 @@ public void populateApkSetBuilder(
 
   @VisibleForTesting
   ImmutableList<Variant> serializeApksForDevice(
-      GeneratedApks generatedApks, DeviceSpec deviceSpec, ApkBuildMode apkBuildMode) {
-    return serializeApks(generatedApks, apkBuildMode, Optional.of(deviceSpec));
+      GeneratedApks generatedApks, DeviceSpec deviceSpec, ApkBuildMode apkBuildMode, Optional<Boolean> debuggable) {
```

Comment:
Could you limit all the lines to 100 chars everywhere please?
https://google.github.io/styleguide/javaguide.html#s4.4-column-limit